src/composables/usePSAccountsContext.ts

```diff
@@ -72,6 +71,9 @@ export default function usePSAccountsContext() {
   const eventbusEnableLink = computed(
     () => state.context.dependencies?.ps_eventbus?.enableLink || '',
   );
+  const eventbusShouldBeInstalled = computed(
```

Comment:
We need the same thing for eventbusShouldBeEnabled because actually we got the message even in 1.6